src/nvim/api/autocmd.c

```diff
@@ -563,6 +489,99 @@ void nvim_del_autocmd(Integer id)
   autocmd_delete_id(id);
 }
 
+/// Clear associated that match the corresponing {opts}. To delete
+/// a particular autocmd, see |nvim_del_autocmd|.
+/// @param opts Parameters
+///         - event: (string|table)
+///              Examples:
+///                 - event: "pat1,pat2,pat3",
```

Comment:
does that work? (it doesn't for `create_autocmd`.)

In any case, I'd prefer not to have two different ways of doing the same thing, so I'd suggest leaving it at
* string for single autocommand
* table for multiple autocommands (with one-key table as trivial but not advertised special case)